tests/cluecode/test_copyrights_fosso.py

```diff
@@ -148,7 +148,7 @@ def build_copyright_test_methods_with_fossology_data():
 def make_test_func(test_file_loc, expected_file_loc, expected):
     def copyright_test_method(self):
         copyrights, _authors, _holders = cluecode_test_utils.copyright_detector(test_file_loc)
-        copyrights = [c.encode('utf-8') for c in copyrights]
+        copyrights = [c for c in copyrights]
```

Comment:
`copyrights = [c for c in copyrights]` does nothing... so this is no longer needed

Reply:
oh sorry,